apps/ui/pages/analytics.vue

```diff
@@ -0,0 +1,119 @@
+<script setup lang="ts">
+interface Trigger {
+  name: string;
+  cssSelector: string;
+  schema: string;
+  app?: string;
+}
+
+interface Resp<T> {
+  data: T[];
+  error: string;
+  status: number;
+}
+
+const devSchema = "pet store";
+
+const { data, error, status } = await useFetch<Resp<Trigger>>(
+  `/${devSchema}/api/trigger`
+);
+
+const showSidePanel = ref(false);
+
+function addTrigger() {
+  showSidePanel.value = true;
+}
+
+async function saveTrigger() {
+  const createAction = {
+    name: formTrigger.value.name,
+    cssSelector: formTrigger.value.cssSelector,
+  };
+  const resp = await $fetch(`/${devSchema}/api/trigger`, {
+    method: "POST",
+    body: JSON.stringify(createAction),
+  });
+  if (resp.status === 200) {
+    clearForm();
+    showSidePanel.value = false;
+  }
+}
+
+function clearForm() {
+  formTrigger.value.name = "";
+  formTrigger.value.cssSelector = "";
+  formTrigger.value.schema = "";
+  formTrigger.value.app = "";
+}
+
+const formTrigger = ref<Trigger>({
+  name: "",
+  cssSelector: "",
+  schema: "",
+  app: "",
+});
+</script>
+
+<template>
+  <Container>
+    <PageHeader title="Analytics" />
+    <ContentBlock
+      class="mt-1"
+      title="Triggers"
+      description="Trigger that fire analytics events"
+    >
+      <Table>
```

Comment:
This is code I would want to use higher order components. I don't want to maintain copies of the code for these kinds of screens multiple times.

In the bootstrap we had 'MolgenisTable' for the table that already has the edit buttons.  
If you must you can use seperate 'RowButtonAdd/Delete/Edit' for the edit dialogues.